_posts/2016-08-05-rgsoc-team-rookies-month-1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Greeting from RGSOC 2016 team Rookies :D
+author: Tehetena & HyeSoo 
+layout: blog-post
+//permalink: /2013/06/Greeting-from-RGSOC-2016-team-Rookies/
+---
+Hi, we’re Tehetena and HyeSoo. We are happy to write about our 2016 summer experience with Hoodie project. We met Hoodie with Rails Girls summer of Code. We joined the Rails Girls summer of Code as a team ‘Rookies’. It has been an amazing experience, since we have started working on the Hoodie desktop application project. 
+
+When we started the project, we had low self confidence. Even though we have taken classes on different programming languages, but when it comes to working with it we always thought that we did not have all the knowledge it took to get the job done. Now we have understood that every working experience is a great opportunity to teach ourselves new things everyday.
+
+The very first lesson we learned was Git. We have done several school projects that we have managed to save and clone project repositories on Git, but after we have started working on the hoodie desktop application, there have been many challenges. At first, we did not know how to propose issues, pull request, fetch from remote, rebase and so on, and we were confused with the concept between local and remote. With our mentor Gregor’s help and many trials & errors, we now have better understanding of how the Github actually works and how it is used in our project and also when many contributors are working on the same project.
```

Comment:
Thank you so muchh :D I changed from Hoodie to hoodie :) and thank you for the congrats. Best wishes for you too.